bolts/data/datasets/audio/ecoacoustics.py

```diff
@@ -257,18 +258,20 @@ def _preprocess_audio(
         if not self._processed_audio_dir.exists():
             mkdir(self._processed_audio_dir)
 
-        waveform_paths = list(self.base_dir.glob("**/*.wav"))
+        waveform_paths = [
+            path for path in self.base_dir.glob("**/*.wav") if "MACOSX" not in str(path)
```

Comment:
```suggestion
            path for path in self.base_dir.glob("**/*.wav") if "MACOSX" not in str(path)
```
Checking each file-name is rather inefficient; why not just glob the EC/UK_BIRD subdirectories directly?

Reply:
Alternatively, you could glob with an exclusion pattern:   
`image_paths.extend(self._base_dir.glob(f"**/[!__]*.{ext}"))`

Reply:
Alternatively, the MACOS directory could be deleted upon download